pkg/client/keycloak/config.go

```diff
@@ -95,6 +100,12 @@ func NewKeycloakConfig() *KeycloakConfig {
 			GrantType:        "client_credentials",
 			Scope:            "api.iam.service_accounts",
 		},
+		InternalSSORealm: &KeycloakRealmConfig{
+			BaseURL:        "https://auth.redhat.com",
+			APIEndpointURI: "/auth/realms/EmployeeIDP",
+			Realm:          "EmployeeIDP",
+		},
```

Comment:
Could this be named something like
```
AdminAPISSORealm
```

and make it configurable? Otherwise this really makes it hard to configure for someone looking to use something different